server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -92,9 +92,10 @@ var (
 )
 
 type BuildEventHandler struct {
-	env           environment.Env
-	statsRecorder *statsRecorder
-	openChannels  *sync.WaitGroup
+	env              environment.Env
+	statsRecorder    *statsRecorder
+	openChannels     *sync.WaitGroup
+	cancelFnsByInvID map[string]func()
```

Comment:
you can use the context.CancelFunc alias here, it gives more clarity to this function

Reply:
Done